saged/models.py

```diff
@@ -1364,17 +1376,32 @@ def fit(self, dataset: LabeledDataset, run: neptune.Run = None) -> "PytorchSuper
 
                 labels = labels.to(device)
 
-                self.optimizer.zero_grad()
-                output = self.model(expression)
+                if type(self.optimizer) == torch.optim.LBFGS:
+                    def closure():
+                        # https://pytorch.org/docs/stable/optim.html#optimizer-step-closure
+                        self.optimizer.zero_grad()
+                        output = self.model(expression)
 
-                loss = self.loss_fn(output, labels)
+                        loss = self.loss_fn(output, labels)
 
-                loss.backward()
+                        loss.backward()
+                        return loss
 
-                if getattr(self, 'clip_grads', False):
```

Comment:
Is gradient clipping not necessary for L-BFGS? Or do you just not want to use it?